docs/airgap-install.md

```diff
@@ -0,0 +1,64 @@
+# Air-gap installation
+
+Air-gap or offline setup is now possible with k0s.
+
+To engage it we use OCI bundle for containerd to provide images to the cluster before even starting kubelet.
+
+There are few ways to achieve offline setup:
+
+- Use OCI bundle with default images
+- Use custom OCI bundle and [override image names](https://docs.k0sproject.io/latest/configuration/#images)
+  through `k0s.yaml`
+- Setup private registry and override image names (not covered by this document)
```

Comment:
maybe add a link to a tutorial how to setup private registry just to guide the users to the optimal solution (at least the one you tested on)